### PR TITLE
fix allergen parsing bug #2719

### LINKED ISSUE #2719: Error from detect_allergens_from_text results in garbled ingredients list
```perl
ingredients_text_fr => "Filet de saumon sauvage certifié MSC, pêché en Pacifique Nord-est (100%)",
```
results in:
```perl
'ingredients_text_with_allergens_fr' => "Filet de saumon sauvage certifi\x{e9} MSC,estp\x{ea}ch\x{e9} en Pacifique Nord- (100%)",
```

### PR DESCRIPTION
**Pull request by stephanegigandet**
Fixes #2719

### CODE REVIEW

File: lib/ProductOpener/Ingredients.pm [hunk 1]
```diff
@@ -3325,7 +3326,7 @@ sub detect_allergens_from_text($) {
 
 			# allergens between underscores
 
-			# print STDERR "current text 1: $text\n";
+			print STDERR "current text 1: $text\n";
```
Comment (VaiTon) [inline lib/ProductOpener/Ingredients.pm]: We shouldn't use STDERR

File: lib/ProductOpener/Ingredients.pm [hunk 2]
```diff
@@ -3344,7 +3345,7 @@ sub detect_allergens_from_text($) {
 
 			# allergens between separators
 
-			#print STDERR "current text 2: $text\n";
+			print STDERR "current text 2: $text\n";
```
Comment (VaiTon) [inline lib/ProductOpener/Ingredients.pm]: We shouldn't use STDERR

File: lib/ProductOpener/Ingredients.pm (hunk 1 above)
Comment (stephanegigandet) [reply lib/ProductOpener/Ingredients.pm]: It's just for debugging/coding, that's why it's commented out. When debugging specific things in the code, I find it much easier than trying to get only the messages I want through log4perl.

Comment (VaiTon) [reply lib/ProductOpener/Ingredients.pm]: I don't think leaving it in the production code would be a good idea. It clutters the console

Comment (stephanegigandet) [reply lib/ProductOpener/Ingredients.pm]: Ah right I didn't intend to leave them, thanks for catching that, sorry I misunderstood the first time.

File: lib/ProductOpener/Ingredients.pm (hunk 2 above)
Comment (stephanegigandet) [reply lib/ProductOpener/Ingredients.pm]: I commented it out. Can you approve the pull request?

**Pull request merged**